commercial/app/views/liveevents/liveEvent.scala.html

```diff
@@ -0,0 +1,31 @@
+@(event: model.commercial.events.LiveEvent, omnitureId: Option[String], clickMacro: Option[String])(implicit request: RequestHeader)
+
+@defining(("2_0", "2016-02-24")) { case (version, date) =>
```

Comment:
I'd leave out the versioning info until someone asks for it.  No-one's ever going to update this anyway so it's redundant.